### PR TITLE
Added page content for screens listed in project spec'.

### PR DESCRIPTION
**Pull request by bernardbaker**
Added page content which was listed in this google doc https://docs.google.com/document/d/1ZfS1JQrqZ4bhy63KcxmfJudwLchiIeXaRBpMaa0tNDo/edit

### CODE REVIEW

**Comment by un7c0rn** [comment]
LGTM

File: src/pages/InstructionsPage.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      <section>
+        <h1>Instructions</h1>
+      </section>
+      {/* This <section> is the width of the view port */}
+      {/* width: 100vw */}
+      <section>
+        {/* These <sections> below should scroll horizontally left and horizontally right - using a swipe motion */}
+        {/* overflow-x: scroll */}
+        <section>
+          <h1>
```
Comment (malinka007) [inline src/pages/InstructionsPage.js]: You are not using any of the MUI elements. Is that intentional due to time constraints?

File: src/pages/components/VoteItem.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    // do something
+    console.log(`vote out team member with ID ${id}`);
+  };
+
+  return (
+    <>
+      <li>
+        <img src={url} alt="profile pic" width="30" height="30" />
+        <p>name</p>
+        <Button ref={buttonRef} label={yourButtonLabel} click={click} />
```
Comment (malinka007) [inline src/pages/components/VoteItem.js]: Why aren't you using Button from Mui?

File: src/pages/InstructionsPage.js (hunk 1 above)
Comment (bernardbaker) [reply src/pages/InstructionsPage.js]: I've left that for you to do. I'm not tasked with doing that.

I simply prepared boilerplate for you.

File: src/pages/components/VoteItem.js (hunk 2 above)
Comment (bernardbaker) [reply src/pages/components/VoteItem.js]: Left that for you. I provided boilerplate code. I wasn't tasked with that. You are. Check the project doc.

My work comes later. TikTok service.

Comment (malinka007) [reply src/pages/components/VoteItem.js]: Gotcha!

**Pull request merged**